src/Apps/Order2/Routes/Checkout/Components/Order2ReviewStep.tsx

```diff
@@ -61,13 +102,25 @@ export const Order2ReviewStep: React.FC<Order2ReviewStepProps> = ({
           </Text>
         </Flex>
       </Message>
+      <Spacer y={4} />
+      {stepState === CheckoutStepState.ACTIVE && (
+        <Button
+          variant={"primaryBlack"}
+          width="100%"
+          onClick={handleClick}
+          loading={loading}
+        >
+          Submit
+        </Button>
+      )}
```

Comment:
minor: I think we should check that our button element renders with a `<button type="submit"/>` attribute by default. This just occurred to me and would probably apply to other buttons on this flow as well.